corehq/form_processor/submission_post.py

```diff
@@ -445,6 +449,35 @@ def do_post_save_actions(case_db, xforms, case_stock_result):
             })
             raise PostSaveError
 
+    @staticmethod
+    def index_case_search(instance, case_stock_result):
+        if not instance.metadata or instance.metadata.deviceID != FORMPLAYER_DEVICE_ID:
+            return
+
+        domain_obj = Domain.get_by_name(instance.domain)
+        if not domain_obj or not domain_obj.web_apps_sync_case_search:
+            return
+
+        from corehq.pillows.case_search import transform_case_for_elasticsearch
+        from corehq.apps.es.case_search import ElasticCaseSearch
+        actions = [
+            BulkActionItem.index(transform_case_for_elasticsearch(case_model.to_json()))
+            for case_model in case_stock_result.case_models
+        ]
+        try:
+            _, errors = ElasticCaseSearch().bulk(actions, refresh=True, raise_on_error=False,
+                                                 raise_on_exception=False)
```

Comment:
Since we're catching and reporting the exception anyways, should `raise_on_*` be `True`?